app/assets/stylesheets/components/_game-card.scss

```diff
@@ -0,0 +1,98 @@
+.game-card{
+  @media (max-width: 768px) {
+    display: none,
+  }
+
+    margin : auto;
+    margin-bottom: 3rem;
+    max-width: 800px;
+    max-height: 220px;
+    display: flex;
+
+
+  .picture{
+    height: 200px;
+    width : 315px;
+  }
+
+  img{
+    object-fit: cover;
+    border-radius: 3px;
+  }
+
+}
+
+.game-content, .game-content-mobile{
+  margin-left: 1rem;
```

Comment:
8 point rule

Reply:
I changed to 16 px :) but isn't the same thing ?